dependencies.yaml

```diff
@@ -66,7 +66,9 @@ files:
       table: project
     includes:
       - depends_on_cupy
-      - depends_on_nvcomp
+      # TODO: restore runtime dependency when we no longer vendor nvcomp
+      # (when nvcomp supports Python 3.13)
```

Comment:
We should probably have a tracking issue for this.

Reply:
Added!